io/pointer/pointer.go

```diff
@@ -73,7 +77,7 @@ type PassOp struct {
 type ID uint16
 
 // Type of an Event.
-type Type uint8
+type Type uint16
```

Comment:
What motivates this change?

Reply:
With this change there are more then 8 pointer Types